src/main/java/com/selenium/course/pages/ChatterPage.java

```diff
@@ -73,6 +71,20 @@ public ChatterPage createAnewPost(String value){
         return clickShareButton();
     }
 
+    public void waitToPublishedPost(){
+        WebElement adTopicsLink = driver.findElement(By.xpath("//a[@class='editLink']"));
+
+        try {
+            wait.withTimeout(5, TimeUnit.SECONDS).until(
```

Comment:
Consider using the TIMEOUT constants created by @caverogustavo 
Apply it to all your pages